lucene/core/src/java/org/apache/lucene/document/FeatureField.java

```diff
@@ -66,8 +66,11 @@
  * 2<sup>-8</sup> = 0.00390625.
  * <p>
  * Given a scoring factor {@code S > 0} and its weight {@code w > 0}, there
- * are three ways that S can be turned into a score:
+ * are four ways that S can be turned into a score:
  * <ul>
+ *   <li>{@link #newLinearQuery w * S}. This is the simplest function
+ *       where no transformation is applied on the feature value, and
+ *       the feature value itself multiplied by weight defines the score.
```

Comment:
"the simplest" might suggest that this would be a good one to start with, when in fact I think this is the most expert function in my opinion given that it expects the feature to already be encoded in the index in a way that makes sense for scoring. Maybe javadocs should better convey that this is expert functionality?